modules/test/playwright/tests/object-web/main/objectView.spec.ts

```diff
@@ -163,7 +163,210 @@ test('cannot create an object custom view using empty multiselectpicklist entry'
 	).toBeVisible();
 });
 
-test('assert that the user is able to use the ERC field in Sort, on the Custom Views tab', async ({
+test('table is updated when changing page on the Custom Object Portlet', async ({
+	apiHelpers,
+	page,
+	viewObjectEntriesPage,
+}) => {
+	const objectDefinitionLabel = 'ObjectDefinitionLabel' + getRandomInt();
+	const objectDefinitionName = 'ObjectDefinitionName' + getRandomInt();
+
+	const {objectEntry, objectFields, titleObjectFieldName} =
+		await mockObjectFields({
+			apiHelpers,
+			objectEntryReturn: {format: 'API'},
+			objectFieldBusinessTypes: ['text'],
+			titleObjectFieldName: 'text',
+		});
+
+	const objectDefinitionAPIClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+	const {body: objectDefinition} =
+		await objectDefinitionAPIClient.postObjectDefinition({
+			active: true,
+			enableLocalization: true,
+			label: {
+				en_US: objectDefinitionLabel,
+			},
+			name: objectDefinitionName,
+			objectFields,
+			pluralLabel: {
+				en_US: objectDefinitionLabel,
+			},
+			portlet: true,
+			scope: 'company',
+			status: {
+				code: 0,
+			},
+			titleObjectFieldName,
+		});
```

Comment:
we can use a simpler function here, `postRandomObjectDefinition()`,  it would be something like: 
```
const objectDefinition =
	await apiHelpers.objectAdmin.postRandomObjectDefinition({
	objectFields,
	objectFolderExternalReferenceCode: 'default',
	status: {code: 0},
	titleObjectFieldName,
});
```